test/format/verify.c

```diff
@@ -234,6 +235,20 @@ table_verify_mirror(
       table->id, checkpoint == NULL ? "" : checkpoint, checkpoint == NULL ? "" : " checkpoint ");
     trace_msg(session, "%s: start", buf);
 
+    /*
+     * If we are not reading from a checkpoint, start a cursor to pin a page to ensure we never
+     * refresh our snapshot in verification.
```

Comment:
Thank you for the comment. I think it needs to be bigger and more detailed about why our snapshot may be refreshed.